src/sentry/static/sentry/app/views/releasesV2/list/clippedHealthRows.tsx

```diff
@@ -43,16 +41,27 @@ class clippedHealthRows extends React.Component<Props, State> {
     const {children, maxVisibleItems, fadeHeight, className} = this.props;
     const {collapsed} = this.state;
 
+    const displayCollapsedButton = !collapsed && children.length > maxVisibleItems;
```

Comment:
I have a suspicion that the `Show XXX More` button does not show the correct amount of projects. Probably something with showing a different number of columns on mobile than on wide?